common/openapi.yml

```diff
@@ -0,0 +1,124 @@
+openapi: 3.0.3
```

Comment:
Why is this file under common ? 

Both status & version are endpoints exposed by a server (service) and subsequently owned by the server. Hence this file should not be tagged as common, but ought to be moved under service folder. 

server - provided the service endpoints (user service). The service has a concept of a version and status. this can exist independent of the client and is deployed on its own
client - generally published as a library and included within other app or services, a client library is not deployable, and hence has no status. the library could have a version, but not exposed as an endpoint. 

Hence not recommended to keep this as a 'common' resource. This is exclusively owned by the service / server

Reply:
When this grows up to be its own service, it will have a client. Then this will be used by both client and server.